src/bootstrap/config.rs

```diff
@@ -974,11 +793,6 @@ impl Config {
 
         config.changelog_seen = toml.changelog_seen;
 
-        let build = toml.build.unwrap_or_default();
```

Comment:
This doesn't look like it needs to be reordered with the profile merging? Can we avoid spurious changes in this file?

Reply:
Yeah, that's useless reorder. Will fix it.